draft-ietf-quic-recovery.md

```diff
@@ -1029,6 +1028,12 @@ SetLossDetectionTimer():
     loss_detection_timer.cancel()
     return
 
+  for (loss_time : loss_times) {
```

Comment:
I don't think this guarantees that you're picking the earliest timer. We need something like 
```
min_loss_time = min_non_zero(loss_time[Initial], loss_time[Handshake], loss_time[ApplicationData])
if (min_loss_time != 0):
    loss_detection_timer.update(min_loss_time)
    return
```

Reply:
Added GetEarliestLossTime(), PTAL